build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'eclipse'
 test {
   jacoco {
     excludes = ['**/package-info**','**/*Test']
-    destinationFile = file("$buildDir/reports/jacoco/test.exec")
+//    destinationFile = file("$buildDir/reports/jacoco/test.exec")
```

Comment:
yes and no, the cloudbees builder is completely broken, for some reason I thought this was the issue, but its an issue with using java6 builder with newer gradle.